frontend/packages/kubevirt-plugin/src/components/vm-templates/vm-template-resource.tsx

```diff
@@ -0,0 +1,77 @@
+import * as React from 'react';
+
+import {
+  getDescription,
+  getOperatingSystemName,
+  getOperatingSystem,
+  getWorkloadProfile,
+  getVmTemplate,
+  getTemplateDisplayName,
+  getFlavor,
+  BootOrder,
+  getBootableDevicesInOrder,
+  TemplateSource,
+} from 'kubevirt-web-ui-components';
+
+import { ResourceSummary } from '@console/internal/components/utils';
+import { DASH } from '@console/shared';
+import { TemplateKind, K8sResourceKind } from '@console/internal/module/k8s';
+
+export const VMTemplateResourceSummary: React.FC<VMTemplateResourceSummaryProps> = ({
+  template,
+}) => {
+  const base = getVmTemplate(template);
+  const baseLink = base && getTemplateDisplayName(base); // TODO(mlibra): link to a template detail, once implemented
```

Comment:
Is this TODO comment still relevant?

Reply:
wow :+1:  , The template details page is getting implemented by this PR ... so we can add the link :-)  here an in the vm details page too ...